snippets/product-compare.liquid

```diff
@@ -53,7 +53,8 @@
                     <span ng-if="value.value" ng-bind="value.value"></span>
                     <span ng-if="!value.value">-</span>
                 </div>
-                <i class="fa fa-check" aria-hidden="true" style="margin: 0 auto;" ng-if="value.valueType.toLowerCase() === 'boolean'"></i>
+                <i class="fa fa-check" aria-hidden="true" style="margin: 0 auto;" ng-if="value.valueType.toLowerCase() === 'boolean' && value.value.toLowerCase() === 'true'"></i>
+                <i class="fa fa-minus" aria-hidden="true" style="margin: 0 auto;" ng-if="value.valueType.toLowerCase() === 'boolean' && value.value.toLowerCase() === 'false'"></i>
```

Comment:
Could we rewrite this in the "if-else" approach?

Reply:
Adding liquid if-else operators in markup does not make it more clear for me personally, this one is ok.